MonsterAttack2/src/components/game/index.jsx

```diff
@@ -0,0 +1,123 @@
+import { useCallback, useState } from "react";
+import { gameConfig } from "../../config/game-configuration";
+import { getRandomNumInInterval } from "../../utils/numbers";
+import ParticipantHealth from "../game-components/participant-health";
+import ActionPanel from "../game-components/action-panel";
+import MovesPanel from "../game-components/moves-panel";
+import PropTypes from "prop-types"
+
+const { player, monster } = gameConfig.participants;
+
+const Game = () => {
+  const [gameStatus, setGameStatus] = useState({
+    currentState: gameConfig.gameStates.notStarted,
+    playerHealth: gameConfig.maxHealth,
+    monsterHealth: gameConfig.maxHealth,
+    playerMoves: [],
+    winner: null
+  });
```

Comment:
extract to defaultStatus = {}
and do - `const [gameStatus, setGameStatus] = useState(defaultStatus)`